runtime/src/bank.rs

```diff
@@ -5013,7 +5007,9 @@ impl Bank {
 
             if let Some((key, count)) = program_to_load {
                 // Load, verify and compile one program.
-                let program = self.load_program(&key, false, None);
+                let program = self
+                    .load_program(&key, false, None)
```

Comment:
Can `load_program` return `None` in any scenario? It seems impossible by looking at the code. Wondering why we changed it to return an `Option`.

Reply:
Yes, if you try to load a nonexistent account. If it does not exist, it has no owner, and thus can not be attributed to a specific loader.

Reply:
Wouldn't that cause the node to crash, given the `expect` in the next line?

Reply:
Nope, because in integration only accounts which have a loader as owner (and thus exist) are selected to be loaded in the first place.